pages/index.js

```diff
@@ -1,23 +1,26 @@
 import { v4 as uuidv4 } from "uuid";
+import Link from "next/link";
 
-import EventCard from "./components/EventCard";
-import Flex from "./components/Flex";
-import { events } from "./api/db.js";
+import EventCard from "../components/EventCard";
+import Flex from "@/components/StyledComponents/Flex";
 
-export default function HomePage() {
+export default function HomePage({ events }) {
   return (
-    <Flex $direction="column">
-      {events.map((event) => {
-        return (
-          <EventCard
-            key={uuidv4()}
-            title={event.title}
-            date={event.date}
-            type={event.type}
-            desc={event.desc}
-          />
-        );
-      })}
-    </Flex>
+    <>
+      <Link href="/create">Event Form</Link>
+      <Flex $direction="column">
```

Comment:
For a flex-direction of column, I'm not sure, why you pass here a prop for that? For now it seems to be a bit over-engineered, since you could give your styled `Flex` component a simple `flex-direction: column`, or?

Reply:
Thanks but I think I will leave it like this for now. It was suggested by another coach.